ca/server.go

```diff
@@ -510,7 +518,11 @@ func (s *Server) Run(ctx context.Context) error {
 			case api.EventDeleteNode:
 				rootReconciler.DeleteNode(v.Node)
 			}
-
+		case <-externalTLSCredsChange:
+			s.signingMu.Lock()
+			s.externalCA.UpdateTLSConfig(NewExternalCATLSConfig(
+				s.securityConfig.ClientTLSCreds.Config().Certificates, s.externalCAPool))
```

Comment:
It seems like this update to `externalCA`'s TLS config is "eventually consistent", since we might pick up events from the `updates` channel between the time the credentials change and this is called. Will that cause any problems?

I suppose the use case here is node certificate renewals, so it seems reasonable to expect the old node certificate to keep working for that short interval. Root rotations might be a bit more of a corner case, but I'm not sure that's worth worrying about either.